lib/pact_broker/client/pacts.rb

```diff
@@ -21,6 +21,11 @@ def publish options
         end
       end
 
```

Comment:
"Get by pact file" doesn't really make sense as a method name. Are you getting by the content or the consumer/provider/version? What you really want to know is if the version has already been published, so let's push that logic down into the method and do:

```ruby
def version_published?

end
```